packages/runtime/lib/app.js

```diff
@@ -248,9 +248,8 @@ class PlatformaticApp {
   }
 
   #setuplogger (configManager) {
-    // Set the logger if not present
     configManager.current.server = configManager.current.server || {}
-    const childLogger = this.#logger.child({}, { level: configManager.current.server.logger?.level || 'info' })
+    const childLogger = this.#logger.child({})
```

Comment:
```suggestion
    const level = configManager.current.server.logger?.level
    let childLogger
    if (level) {
      childLogger = this.#logger.child({ level })
    } else {
      childLogger = this.#logger.child({})
    }
```

I'm actually partially convinced that you could skip creating the child logger altogether if there is no level set.

Reply:
1. How do you skip creating the child logger in your suggestion?
2. If the child log level is `info` and runtime log level is `trace`, it will send a trace logs to the control/meraki.

Reply:
1. you can just set `this.#logger` inside the config 
2. exactly. It's the user choice.